addons/website_slides_survey/static/src/js/slides_upload.js

```diff
@@ -22,8 +22,9 @@ SlidesUpload.SlideUploadDialog.include({
     * Will automatically set the title of the slide to the title of the chosen certification
     */
     _onChangeCertification: function (ev) {
-        if (ev.added && ev.added.text) {
-            this.$("input#name").val(ev.added.text);
+        let $inputName = this.$("input#name");
```

Comment:
`const $inputElement` ?

"inputName" is a bit confusing as it makes you think it's the value.
(You can use "const" here as you are not modifying the variable).